### ISSUE TITLE
CT: Caution flag erroneously showing for Academy College

### ISSUE THREAD

**Issue by desireeturner** [department-of-veterans-affairs/va.gov-team#14523]
## What happened?
TLDR: Academy College is in good standing as far as accreditation goes.  We show a caution flag and we should not.  We are not sure what in our data is generating the caution flag.

Details: 
Let me try to sum it up.  We have a school, Academy College, that shows a caution flag on the GIBCT for accreditation.  That is something that comes from the Dept of Ed.  Looking at their files in GIDS they do show a loss of accreditation in Jan 2019 due to a voluntary withdrawal.  This is in the accreditationaction file.  But they have an active accreditation on file with Dept of Ed and that shows in the accreditationrecord file.  If you go to the Dept of Ed site it shows they have an active accreditation. 

We were trying to figure out what was going on here.  I pulled another school, Vantage Career Center.  In the accreditationaction file they show an accreditation that expired in June 2020.  IN the accreditationrecord file they should an active accreditation.  Dept of Ed shows them active.  And in this case we show they are good on the GIBCT, no caution flag.

Brian and I were on the phone for a while today trying to figure this out and we could not. 

Academy College dapipid 123396
Vantage Career Center dapipid 162326

![Screen Shot 2020-10-07 at 9.57.13 AM.png](https://images.zenhubusercontent.com/5b645f6fe2255460d8f0b032/914d69e7-c61c-4faf-abc6-936eb07b7076)
![Screen Shot 2020-10-07 at 9.57.13 AM.png](https://images.zenhubusercontent.com/5b645f6fe2255460d8f0b032/a40b53e4-399e-4ca2-8e85-2745c63fc183)

## Steps to Reproduce
- URL: https://www.va.gov/gi-bill-comparison-tool/
1. Search for "Academy College". View caution flag for Academy College related to accreditation 

## Desired behavior
1. Academy College should not be displaying an accreditation caution flag

## Acceptance Criteria
1. Academy College does not display an accreditation caution flag

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to Epic** (what body of work is this a part of? possibly `Ongoing Maintenance`)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with `Bug`**

**Comment by shawkey-daniel** [comment]
Works as designed. The accreditation action file reports the status as "Loss of Accreditation or Preaccreditation: Voluntary Withdrawal". This will generate a caution flag. The flag will be disabled when a new accreditation action with a restorative status is added.

**Comment by zurbergram** [comment]
Investigating using this data
![image](https://user-images.githubusercontent.com/1094999/95359645-65bf0200-0898-11eb-8a57-dfd2357f13ce.png)


[AccreditationInstituteCampus.txt](https://github.com/department-of-veterans-affairs/va.gov-team/files/5342216/AccreditationInstituteCampus.txt)
[AccreditationAction.txt](https://github.com/department-of-veterans-affairs/va.gov-team/files/5342272/AccreditationAction.txt)
[AccreditationRecord.txt](https://github.com/department-of-veterans-affairs/va.gov-team/files/5342215/AccreditationRecord.txt)

**Comment by zurbergram** [comment]
Academy College dapipid 123396 has 4 rows in AccreditationAction.

The 3rd row, date of 2018-12-21, has an `action_description` that has not received a restorative status

```
123396,16,ACCREDITING COUNCIL FOR INDEPENDENT COLLEGES AND SCHOOLS,1,Institutional Accreditation,1,Renewal of Accreditation,2014-04-11,Is in compliance with all of the agency's accreditation standards,,
123396,16,ACCREDITING COUNCIL FOR INDEPENDENT COLLEGES AND SCHOOLS,1,Institutional Accreditation,1,Removal of Show Cause Status,2018-05-22,Has satisfactorily addressed all compliance concerns from previous actions,,
123396,16,ACCREDITING COUNCIL FOR INDEPENDENT COLLEGES AND SCHOOLS,1,Institutional Accreditation,1,Probation or Equivalent or a More Severe Status: Warning,2018-12-21,Fails to meet agency standards for initial or renewed accreditation,,
123396,16,ACCREDITING COUNCIL FOR INDEPENDENT COLLEGES AND SCHOOLS,1,Institutional Accreditation,1,Loss of Accreditation or Preaccreditation: Voluntary Withdrawal,2019-01-29,Other - Provide unlisted or multiple justifications,Voluntarily withdrew,
```

**Comment by desireeturner** [comment]
This is working as designed.
